lib/std/os/bits/wasi.zig

```diff
@@ -271,11 +276,33 @@ pub const linkcount_t = u64;
 pub const lookupflags_t = u32;
 pub const LOOKUP_SYMLINK_FOLLOW: lookupflags_t = 0x00000001;
 
-pub const oflags_t = u16;
-pub const O_CREAT: oflags_t = 0x0001;
-pub const O_DIRECTORY: oflags_t = 0x0002;
-pub const O_EXCL: oflags_t = 0x0004;
-pub const O_TRUNC: oflags_t = 0x0008;
+pub usingnamespace if (builtin.link_libc) struct {
+    pub const O_ACCMODE = (O_EXEC | O_RDWR | O_SEARCH);
+    pub const O_APPEND = 1 << 0; // = __WASI_FDFLAGS_APPEND
+    pub const O_CLOEXEC = (0);
+    pub const O_CREAT = ((1 << 0) << 12); // = __WASI_OFLAGS_CREAT << 12
+    pub const O_DIRECTORY = ((1 << 1) << 12); // = __WASI_OFLAGS_DIRECTORY << 12
+    pub const O_DSYNC = (1 << 1); // = __WASI_FDFLAGS_DSYNC
+    pub const O_EXCL = ((1 << 2) << 12); // = __WASI_OFLAGS_EXCL << 12
+    pub const O_EXEC = (0x02000000);
+    pub const O_NOCTTY = (0);
+    pub const O_NOFOLLOW = (0x01000000);
+    pub const O_NONBLOCK = (1 << 2); // = __WASI_FDFLAGS_NONBLOCK
```

Comment:
Is it possible to do

```zig
pub const O_NONBLOCK = FDFLAG_NONBLOCK;
```

And for any other suitable occurrence?

Reply:
done: https://github.com/ziglang/zig/pull/9227/commits/7814a2bd4a3ec22cd9548c622f7dc837dba968f7